docs/datasets/stack.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>I think we need to refer to a general page showing the formula. I suppose that irf theory is right place for that. </p><p>That's for another future PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3282/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>